.github/workflows/previewPR.yml

```diff
@@ -107,10 +127,18 @@ jobs:
           source: architectures/index.scss
           destination: architectures/index.css
           
+      - name: Compile sass for tutorials
+        uses: gha-utilities/sass-build@v0.4.2
+        with:
+          source: wiki-tutorials/index.scss
+          destination: wiki-tutorials/index.css
+          
       - name: Remove files that are breaking the build
         run: |
           rm architectures/README.md || true
           rm architectures/index.scss || true
```

Comment:
Do we need changes here as of rename of repo? @SchettlerKoehler ?

Reply:
It would be clean to change it but it is not needed for a working workflow